tests/integration/targets/mso_tenant_site/tasks/main.yml

```diff
@@ -286,6 +292,7 @@
   assert:
     that:
     - aazure_shared_nm is not changed
+  when: version.current.version is version('4.0', '<') # TODO likely due to header behaviour changed, needs further investigation
```

Comment:
can you recheck this one?

Reply:
changed with true so need the version check

Reply:
Was this one about the changed value not being reflected correct on Tenant PATCH? Did we ask Ty about this one?

Reply:
yes that is this one, and not yet